config/webpack-config.js

```diff
@@ -66,8 +66,19 @@ var onDemandConf = {
   ]
 };
 
+var nonResponsiveConf = {
+  context: path.join( __dirname, '/../', paths.unprocessed,
+                      JS_ROUTES_PATH + '/on-demand' ),
+  entry:   './header.js',
+  output: {
+    path:     path.join( __dirname, 'js' ),
+    filename: '[name]'
+  }
+};
+
 module.exports = {
-  onDemandConf: onDemandConf,
-  ieConf:       ieConf,
-  modernConf:   modernConf
+  nonResponsiveConf: nonResponsiveConf,
```

Comment:
`onDemandHeaderRawConf`? Kinda a mouthful, but not terrible. At any rate, aiming to make it more descriptive of what it does vs where it's used would be good, that way if it ends up being used somewhere else it'll still make sense.